### DISCUSSION TITLE
Welcome to FlashFloppy Discussions!

### DISCUSSION THREAD

**Discussion by keirf** [keirf/flashfloppy#404] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
 
  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DarrellH89** [comment]
Hi,

I'm using Flash Floppy on a Heathkit H-89. I have an HxC drive on a second system. Flash Floppy works fine with two bootable images in HFE format. I'm having trouble setting parameters for the IMG format. I took the values from the IMD file.

So far I've got:
##################################################
[::819200]
cyls = 80
heads = 2
interleave = 3
secs = 5
bps = 1024
rate = 250
##################################################
[H37::655360]
cyls = 80
heads = 2
interleave = 3
secs = 16
bps = 256
rate = 250

Thanks for any help you are able to provide.

  Darrell

**Comment by Papalapaa** [comment]
Hi there,

I use flashfloppy 3.23 with an Amiga 500 and I'd like to know if is possible to "load" more than one ADF file at the same time on the slots. I mean, to copy several ADF files from the USB to the slots in a batch instead of copying them one bye one.

Many thanks :)
